steps/steps.py

```diff
@@ -0,0 +1,96 @@
+from behave import *
+from selenium import webdriver
+
+
+import environment
+import time
+
+
+class Calculator(object):
+    @step('that the Calculator app is loaded')
+    def app_is_loaded(context):
+        context.browser.get(
+            'http://seleniumsimplified.com/testpages/calculate.php'
+        )
+
+    @step('there is no current value in the calculator')
+    def check_current_value(context):
+        first_number_input = context.browser.find_element_by_xpath(
+            '//input[@id="number1"]'
+        )
+        second_number_input = context.browser.find_element_by_xpath(
+            '//input[@id="number2"]'
+        )
+
+        if first_number_input.get_attribute("value") != '':
+            first_number_input.clear()
+            if second_number_input.get_attribute("value") != "":
+                second_number_input.clear()
```

Comment:
Jeśli tak zagnieżdżasz `if`y, to nie wejdziemy w drugi `if` jeśli pierwszy będzie fałszem. Czyli jeśli pierwszy input będzie pusty, a drugi "pełny", to ostatecznie drugi nie zostanie wyczyszczony. Nieraz zagnieżdżenie ma sens, ale tutaj w zupełności wystarczy jeden poziom, czyli:

    if first_number_input.get_attribute("value") != '':
        first_number_input.clear()
    if second_number_input.get_attribute("value") != "":
        second_number_input.clear()

Co więcej, tego typu porównania (czy coś nie jest pustym stringiem, zerem, pustą listą itp.) robi się poprzez samo sprawdzenie czy istnieje, czyli "more pythonic" będzie tak:

    if first_number_input.get_attribute('value'):
        first_number_input.clear()
    if second_number_input.get_attribute('value'):
        second_number_input.clear()